a3p-integration/.gitignore

```diff
@@ -15,6 +15,7 @@ upgrade-test-scripts
 # same for each proposal, an independent project
 proposals/*/.pnp.*
 proposals/*/.yarn/*
+proposals/a:upgrade-next/*
```

Comment:
That's definitely too broad. Please revert and use a targeted ignore file inside the `proposals/a:upgrade-next` folder